code/game/objects/items/cigs_lighters.dm

```diff
@@ -933,7 +933,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	worn_icon_state = "vape_worn"
 	greyscale_config = /datum/greyscale_config/vape
 	greyscale_config_worn = /datum/greyscale_config/vape/worn
-	greyscale_config_worn_digi = /datum/greyscale_config/vape/worn/digi
+	greyscale_config_worn_digi = /datum/greyscale_config/vape/worn/digi  //SKYRAT EDIT - DigiGreyscale (Digi? On a vape? What? How did we do this)
```

Comment:
Just keep the `ADDITION` in there, so it's clear this wasn't an upstream thing (because it isn't from upstream :) )
```suggestion
	greyscale_config_worn_digi = /datum/greyscale_config/vape/worn/digi  //SKYRAT EDIT ADDITION - DigiGreyscale (Digi? On a vape? What? How did we do this)
```

Reply:
Yuh I'll get it in - I was adding "DigiGreyscale" so in the future if/when its all changed or made modular we can search one phrase to see all the nonmodular ones